style.css

```diff
@@ -1 +1,3 @@
-html {}
+html {
+  background-color: white;
+}
```

Comment:
You should style your CSS file. The CSS file is to be used to style your HTML page, and not using it will make your page less pleasing to the person who accesses it.